api.yml

```diff
@@ -17,13 +17,38 @@ tags:
   - name: public
     description: Public APIs facing the Holder and Verifier.
 paths:
-  /credential:
+  /credentials:
     post:
       tags:
         - internal
       summary: Issues a credential and returns it in the response body.
       operationId: issueCredential
       description: Issues a credential and returns it in the response body. Support of this part of the API is REQUIRED for implementations.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IssueCredentialRequest'
+        description: Parameters for issuing the credential.
+      responses:
+        '201':
+          description: Credential successfully issued!
+          content:
+            application/json:
+              schema:
+                type: object
+                description: The issued credential. This SHOULD include a reference or ID that can be used later for other parts of the API, e.g. changing and retrieving the status of the credential, or refreshing the credential.
+        '400':
+          description: invalid input!
+        '500':
+          description: error!
+  /credential:
```

Comment:
Both operations should be under a common path.

Reply:
I disagree... but if others agree, then we could do .../credentials/compositions.

What I would rather we do is .../compositions and .../credentials.

I'll reiterate that Digital Bazaar objects to the compositions API as defined in the current document (as well as a subset of the other endpoints that we're keeping around).

Reply:
Resolving because this is now outdated.